src/packages/core/property/property-dataset/property-dataset.element.ts

```diff
@@ -0,0 +1,121 @@
+import type { UmbPropertyValueData } from '../types/property-value-data.type.js';
+import { UmbPropertyDatasetBaseContext } from './property-dataset-base-context.js';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { customElement, html, property } from '@umbraco-cms/backoffice/external/lit';
+import { UmbChangeEvent } from '@umbraco-cms/backoffice/event';
+
+/**
+ *  @element umb-property-dataset
+ *  @description - Element for hosting a property dataset. This is needed for umb-property to work.
+ *  @slot default - Slot for rendering content within.
+ */
+@customElement('umb-property-dataset')
+export class UmbPropertyDatasetElement extends UmbLitElement {
+	// A take on only firing events when the value is changed from the outside.
+	#silentOnce = true;
```

Comment:
What is the use case of this? It goes against the nature of events that it is when changes happen inside a system to notify the outside. The outside should already know what the value is. 

Maybe we can find a different approach to solve the use case.

Reply:
Yeah, i was a bit torn by this one. Cause we usually only fire events when the user did some interaction. But in this case we are too far from interaction for me to know wether its interaction or just caused by usual data change. So this is a fix, so when setting data via example the props it does not trigger an update. In other words, If i (as the implementation) put new data, I dont want a event that there is new data? Maybe lets talk about this in a huddle?

Reply:
We have implemented some corrections from a huddle talk we had.